prelude/bootstrap.js

```diff
@@ -1243,7 +1243,8 @@ function payloadFileSync(pointer) {
   }
 
   function revertMakingLong(f) {
-    if (/^\\\\\?\\/.test(f)) return f.slice(4);
+    const prefixRegex = /^\\\\\?[\\]+/
```

Comment:
May be prefixRegex could be put in global scope to spare the time to construct the Regexp machinery each time.

Reply:
Right!

Reply:
Or I could make it part of the function